installers/pgo-installer/ansible/roles/pgo-installer/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+pgo_installer_namespace: "pgo-install"
+pgo_install_sa: "pgo-installer-sa"
+pgo_install_crb: "pgo-installer-crb"
+job_timeout: -1s
```

Comment:
need to change to a reasonable timeout for the install job to complete. I'm thinking 120s-180s but feedback welcomed